gems/smithy-xml/spec/spec_helper.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require 'simplecov'
+SimpleCov.start do
+  add_filter '/spec/'
+  add_filter 'gems/smithy/'
+  add_filter 'gems/smithy-cbor/'
+  add_filter 'gems/smithy-client/'
+  add_filter 'gems/smithy-json/'
+  add_filter 'gems/smithy-schema/'
+end
+
+require 'smithy-xml'
```

Comment:
Most of the actual testing will be protocol tests but probably should cover similar test cases that we have in place for smithy-json.